### ISSUE TITLE
Darktable not loading under latest Fedora 29 package update

### ISSUE THREAD

**Issue by JohnBeeR1** [darktable-org/darktable#2137]
**Describe the bug**
Darktable fails to load. No problems running last week, software updates ran at weekend, not including Darktable, so I suspect the problem lies in something else that was released.

**To Reproduce**
Running Darktable under gdb also hangs and has to be interrupted. Backtrace in 'Screenshots'.

**Expected behavior**
Darktable to load

**Screenshots**
 
[gdb.txt](https://github.com/darktable-org/darktable/files/2910285/gdb.txt)

**Platform (please complete the following information):**
 - OS: Fedora 29
 - Version  Version 3.30.2

**Additional context**

**Comment by aurelienpierre** [comment]
What version of dt are you using ? A packaged 2.6.0 or a self-compiled master ? 

I run on Fedora 29 myself, I have never experienced such issues. If it affects a package, you should contact the packagers.

**Comment by JohnBeeR1** [comment]
I have also not had any problems running under 29 until this week, but I also can't see from the backtrace where the problem lies.
I'm running the fedora package which is 2.6.0-2.  I've removed it and reinstalled.  I've tried running with the older Gnome boot that I was running last week, but still a problem.  It has to be a problem with another package that updated at the weekend, but no idea which one.
